frontend/src/pages/DataCatalog/DataCatalogPage.tsx

```diff
@@ -58,23 +56,33 @@ export default function DataCatalogPage() {
       <Helmet>
         <title>Data Downloads - Health Equity Tracker</title>
       </Helmet>
-      <h2 className='sr-only'>Data Downloads</h2>
-      <div className='mx-auto min-h-full max-w-md flex-col p-10'>
-        <header className='pb-2'>
-          <h3
-            id='main'
-            className='m-0 font-serif text-smallHeader font-light leading-lhSomeSpace text-altBlack'
-          >
-            View and download Health Equity Tracker data sources
-          </h3>
-          <p className='text-small'>
-            Here you can access and download the data sources that are displayed
-            in the charts on the Health Equity Tracker. Want to explore what
-            each data set can show us about different health outcomes?{' '}
-            <a href={EXPLORE_DATA_PAGE_LINK}>Explore the data dashboard</a>
-            <span aria-hidden={true}>.</span>
-          </p>
-        </header>
+      <section
+        id='main-content'
+        aria-labelledby='main-content'
+        tabIndex={-1}
```

Comment:
this `<section>` isn't interactive so i don't think it needs any type of label or tab index; i think we can remove both lines? Also, it doesn't make sense to have the aria labelled by refer to its own id; perhaps it could point to `'main'` but even still i dont think you should aria label a container like section